guidance/library/_set_var.py

```diff
@@ -0,0 +1,21 @@
+import guidance
+from ._block import block
+
+@guidance
+def set_opener(lm, name, value):
+    if name in lm:
+        lm = lm.set("__save" + name, lm[name])
+    return lm.set(name, value)
+
+@guidance
+def set_closer(lm, name):
+    if "__save" + name in lm:
+        return lm.set(name, lm["__save" + name]).remove("__save" + name)
+    else:
+        return lm.remove(name)
+
+def set_var(name, value=True):
```

Comment:
I see you implemented this approach to set the same values as in the `set_attribute` method you did for blocks, but I don't see it used anywhere for now. Is this something you're planning to use later on?

Reply:
Ah, I just checked your commit history and you were using variables instead of attributes for the blocks before, so maybe a forgotten file, unless you think it's still useful for other purposes?